sleap_anipose/calibration.py

```diff
@@ -351,6 +352,75 @@ def write_board_cli(
     )
 
 
+def draw_board(
+    save_folder: str,
+    board_X: int,
+    board_Y: int,
+    square_length: float,
+    marker_length: float,
+    marker_bits: int = 4,
+    dict_size: int = 1000,
+):
+    """Draw and save a printable calibration board jpg file.
+
+    Args:
+        save_folder: Path to save the file to.
+        board_X: Number of squares along the width of the board.
+        board_Y: Number of squares along with height of the board.
+        square_length: Length of square edges in meters.
+        marker_length: Length of marker edges in meters.
+        marker_bits: Number of bits in aruco markers, default 4.
+        dict_size: Size of dictionary for encoding aruco markers, default 1000.
+    """
+    # Add more options for aruco dictionaries later.
+    if marker_bits == 4:
+        if dict_size == 1000:
+            aruco_dict = cv2.aruco.Dictionary_get(cv2.aruco.DICT_4X4_1000)
+    else:
+        aruco_dict = cv2.aruco.Dictionary_get(cv2.aruco.DICT_4X4_1000)
+
+    charuco_board = cv2.aruco.CharucoBoard_create(
+        board_X, board_Y, square_length, marker_length, aruco_dict
+    )
+    imboard = charuco_board.draw((1080, 1080))
```

Comment:
What are these numbers?? should they be function arguments?

Reply:
these correspond to the number of pixels in the image along each dimension. Do you prefer to have image size as a user parameter?

Reply:
yes -- no [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) in code!!

Reply:
will add these as parameters

Reply:
resolved